pkgs/development/libraries/libgudev/default.nix

```
@@ -18,11 +21,11 @@ stdenv.mkDerivation rec {
     sha256 = "094mgjmwgsgqrr1i0vd20ynvlkihvs3vgbmpbrhswjsrdp86j0z5";
   };
 
-  nativeBuildInputs = [ pkg-config gobject-introspection ];
-  buildInputs = [ udev glib ];
+  nativeBuildInputs = [ pkg-config gobject-introspection meson ninja ];
+  buildInputs = [ udev glib vala ];
 
   # There's a dependency cycle with umockdev and the tests fail to LD_PRELOAD anyway.
-  configureFlags = [ "--disable-umockdev" ];
+  mesonFlags = [ "-Dtests=disabled" ];
```

While at it, we can switch to the cleaner one dep/flag per line.

```suggestion
  mesonFlags = [
    # There's a dependency cycle with umockdev and the tests fail to LD_PRELOAD anyway.
    "-Dtests=disabled"
  ];
```